workspace/locales/en.yml

```diff
@@ -42,6 +42,11 @@ en:
             desc: "Lists all artifacts of latest build"
             syntax_specific: "*@buildbot artifacts `build_name` `build_number`*"
             desc_specific: "Lists all artifacts of build_number"
+          success_today:
+            syntax: "*@buildbot success_today*"
+            desc: "Lists all success builds today"
+            syntax_wild: "*@buildbot success_today `some_text*`*"
+            desc_wild: "Lists success builds today matching a case-sensitive wildcard search with `*`"
```

Comment:
Lists successful builds from today for a `build_id`.`

Reply:
Please also refactor the descriptions for `running` and `queue` commands.  They should be `build_id`, not `some_text*`, correct?

Reply:
@scopp you want 'build_id' or 'build_id*'? because '*' stands for "and any text after build_id". If we remove it here then we should remove it in the route as well.